README.md

```diff
@@ -139,7 +139,7 @@ To connect your hosting server with your repo is some configuration needed.
 
 Add something like this to your configuration file:
 
-```shell
+```php
```

Comment:
@dev7ch does not look like php commands...

Reply:
https://luya.io/guide/module/luyadev---luya-deployer#hosts-and-deployment-key

Reply:
doenst work with ```shell

Reply:
but its not php. use `sh` instead